app/views/user/edit_mute.haml

```diff
@@ -0,0 +1,4 @@
+= render 'settings/muted', qr_svg: @qr_svg, recovery_code_count: @recovery_code_count
```

Comment:
qr_svg?

Reply:
Wow, I blindly duplicated the line from Security.